src/hacking/editor-support.md

```diff
@@ -53,6 +53,10 @@ To enable [optional build settings](building-servo.md#optional-build-settings),
 }
 ```
 
+### Windows Users
+
+If you are on Windows, you will need to use `./mach.bat` instead of just `./mach`. Not doing so will cause rust-analyzer to throw an error when running any of these commands.
```

Comment:
can we include the actual error below, to help people search for it? see [§ Error messages](https://book.servo.org/style-guide.html#error-messages) for how to do that, also one sentence per line please :)